homeassistant/components/fivem/__init__.py

```diff
@@ -103,42 +98,35 @@ async def initialize(self) -> None:
         info = await self._fivem.get_info_raw()
         self.server = info["server"]
         self.version = info["version"]
-        self.gamename = info["vars"]["gamename"]
+        self.game_name = info["vars"]["gamename"]
 
     async def _async_update_data(self) -> dict[str, Any]:
         """Get server data from 3rd party library and update properties."""
-        was_online = self.online
-
         try:
             server = await self._fivem.get_server()
             self.online = True
         except FiveMServerOfflineError:
             self.online = False
 
-        if was_online and not self.online:
-            _LOGGER.warning("Connection to '%s:%s' lost", self.host, self.port)
-        elif not was_online and self.online:
-            _LOGGER.info("Connection to '%s:%s' (re-)established", self.host, self.port)
-
-        if self.online:
-            players_list: list[str] = []
-            for player in server.players:
-                players_list.append(player.name)
-            players_list.sort()
+        if not self.online:
+            raise UpdateFailed
 
-            resources_list = server.resources
-            resources_list.sort()
+        players_list: list[str] = []
+        for player in server.players:
+            players_list.append(player.name)
+        players_list.sort()
 
-            return {
-                NAME_PLAYERS_ONLINE: len(players_list),
-                NAME_PLAYERS_MAX: server.max_players,
-                NAME_RESOURCES: len(resources_list),
-                NAME_STATUS: self.online,
-                ATTR_PLAYERS_LIST: players_list,
-                ATTR_RESOURCES_LIST: resources_list,
-            }
+        resources_list = server.resources
+        resources_list.sort()
 
-        raise UpdateFailed
+        return {
+            NAME_PLAYERS_ONLINE: len(players_list),
+            NAME_PLAYERS_MAX: server.max_players,
+            NAME_RESOURCES: len(resources_list),
+            NAME_STATUS: self.online,
```

Comment:
We don't really need to return this explicitly since the coordinator already has an attribute for it called `last_update_success`.
```suggestion
            NAME_STATUS: self.last_update_success,
```